controllers/ControllerAdmin.php

```diff
@@ -3,201 +3,212 @@
 
-		$emailExist = $this->_userManager->checkEmail($params['identifiant']);
+		$userManager = (new AuthManager);
+		$emailExist = $userManager->checkEmail($params['identifiant']);
 
 		if ($params['identifiant'] != $key['email'] && $emailExist == true)
 		{
-			$noEmail = true;
-			echo $this->_twig->render('profile.twig', ['noEmail' => $noEmail,
-			                                           'server' => $this->_server,
-			                                           'logged'=> TRUE,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+			echo $this->getTwig()->render('profile.twig', ['noEmail' => TRUE,
+			                                               'server' => $this->getServer(),
+			                                               'logged'=> TRUE,
+			                                               'user'=> $key['pseudo'],
+			                                               'value'=> $key['email'],
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 			exit();
 		}
 
 		elseif ($params['identifiant'] === $key['email'])
 		{
-			echo $this->_twig->render('profile.twig', ['logged'=> TRUE,
-			                                           'server' => $this->_server,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+			echo $this->getTwig()->render('profile.twig', ['logged'=> TRUE,
+			                                               'server' => $this->getServer(),
+			                                               'user'=> $key['pseudo'],
+			                                               'value'=> $key['email'],
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 		}
 
 		else
 			{
-			$validChange = $this->_userManager->updateUserMail($params['identifiant'], $key);
+			$validChange = $userManager->updateUserMail($params['identifiant'], $key);
+
 			if ($validChange){
 				(new Session)->setValueKey('user', 'email', $params['identifiant']);
 				$key['email'] = $params['identifiant'];
-				echo $this->_twig->render('profile.twig', ['confirm' => TRUE,
-				                                           'server' => $this->_server,
-				                                           'logged' => TRUE,
-				                                           'user' => $key['pseudo'],
-				                                           'value' => $key['email'],
-				                                           'avatar'=> $key['avatar']]);
+
+				echo $this->getTwig()->render('profile.twig', ['confirm' => TRUE,
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)